src/borg/remote.py

```diff
@@ -540,6 +540,7 @@ def __init__(self, location, create=False, exclusive=False, lock_wait=None, lock
         self.unpacker = get_limited_unpacker('client')
         self.server_version = parse_version('1.0.8')  # fallback version if server is too old to send version information
         self.p = None
+        self._args = args
```

Comment:
Needed access to arguments in `ssh_cmd` to replace using the global `os.environ`.